src/browser-runner/platform-dummy/sagas/server-command-handlers.js

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+import { takeEvery } from 'redux-saga/effects';
+import opn from 'opn';
+
+import SharedActions from '../../../shared/actions/shared-actions';
+
+function create({ payload: { url } }) {
+  console.log(`Chrome frontend hosted at ${url}`); // eslint-disable-line no-console
+  opn(url);
```

Comment:
This is really smart! If we'd supposedly have a "new window" button in the chrome, clicking on it while the frontend runs in a browser tab would automagically spawn a new tab with a new frontend window. Very nice.